xpk.py

```diff
@@ -1933,6 +1931,56 @@ def get_gke_outlier_dashboard(args):
 
   return None
 
+def get_gke_outlier_dashboard(args):
+  """Get the identifier of GKE outlier dashboard deployed in the project.
+
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    str:
+      identifier of outlier dashboard if deployed in project,
+      None otherwise.
+  """
+  outlier_dashboard_filter = "displayName:'GKE - TPU Monitoring Dashboard'"
+  dashboard_id = get_gke_dashboard(args, outlier_dashboard_filter)
```

Comment:
Overall looks great!

I see that we check the dashboard_id in two ways before erroring:
1) if it is None which behind the scenes means an error in the command
2) If not dashboard_id which means the dashboard id is empty and there are no dashboards available.

I think we can make this easier to read by having get_gke_dashboard return a tuple with (return_error, return_val) syntax? what do you think about this?